components/TextField/TextField.tsx

```diff
@@ -48,19 +51,11 @@ const TextField: FC<TextFieldTypeProps> = ({
         disabled={disabled}
         onChange={onChange}
       />
+      {variant === "error" && (
+        <span className={styles.textField_errorMessage}>Your input format is incorrect</span>
```

Comment:
question: What if you want to reuse this TextField in multiple places with different error messages? Maybe it should be a prop, eg `error`